src/core/etl/src/Flow/ETL/Row/Entry/Type/Type.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\ETL\Row\Entry\Type;
```

Comment:
I'm not 100% sure if that namespace is the correct one. I think I would take it even higher, to `\Flow\ETL\PHP\Type` maybe since those objects are representing PHP Types

Reply:
Moved there.